botogram/components.py

```diff
@@ -158,6 +160,22 @@ def add_message_edited_hook(self, func):
         hook = hooks.MessageEditedHook(func, self)
         self.__messages_edited_hooks.append(hook)
 
+    def add_channel_post_hook(self, func):
+        """Add a channel post hook"""
+        if not callable(func):
+            raise ValueError("A channel post hook must be callable")
+
+        hook = hooks.ChannelPostHook(func, self)
+        self.__channel_post_hooks.append(hook)
+
+    def add_edited_channel_post_hook(self, func):
```

Comment:
Maybe it's better to call this `add_channel_post_edited_hook`, for the sake of consistency with `add_message_edited_hook`.